cmd/ib-sriov-cni/main.go

```diff
@@ -32,18 +35,20 @@ func init() {
 func cmdAdd(args *skel.CmdArgs) error {
 	netConf, err := config.LoadConf(args.StdinData)
 	if err != nil {
-		return fmt.Errorf("InfiniBand SRI-OV CNI failed to load netconf: %v", err)
+		return fmt.Errorf("infiniBand SRI-OV CNI failed to load netconf: %v", err)
```

Comment:
Why that change? and it that from linter berterr to write inifiniband

Reply:
IIRC that came from golint.
in general the object generated by fmt.Errorf(..), its string interpretation cannot begin with uppercase nor end with period.

i guess its because someone may use the string in the middle of a sentence/ add period/colon